datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -201,8 +201,9 @@ def check_duplicate_values(current_check, line, row, header_name, duplicates, fa
 @click.option(
     '--check-duplicates', is_flag=True, help='Output warnings if there are duplicate short names and descriptions'
 )
+@click.option('--hide-warnings', '-h', is_flag=True, help='Hide warnings and only show failures')
```

Comment:
Would the reverse be better?  To hide warnings by default, but have an option `show-warnings` to see them?

Reply:
Sure, that works!